src/phpDocumentor/Transformer/Writer/Twig/LinkRenderer.php

```diff
@@ -62,6 +64,9 @@ public function __clone()
         $this->adapters = $this->createAdapters();
     }
 
+    /**
+     * @deprecated will be removed once getProject is removed
```

Comment:
I'm not sure if we should remove the full project from the rendering... You might need the project in some cases.
For example when you need a list of versions in a template.

Reply:
As far as I am concerned, we can reintroduce that at that moment. Partly because I am not sure whether this specific way of handling will be preferable or how linking will come together.